book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
```

Comment:
@rentals should also have a Getter method, therefore it should be set under the `attr_accessor` instead.